requirements-dev.in

```diff
@@ -11,3 +11,4 @@ pygments
 types-six
 types-python-dateutil
 types-PyYAML
+standard-imghdr
```

Comment:
> [!NOTE]
> The `imghdr` module was removed from the standard library in Python 3.13. This module is required by the version of Sphinx we use (v4.3.2). This issue was [addressed in Sphinx-6.2.0](https://github.com/sphinx-doc/sphinx/issues/10440#issuecomment-1556180835). Because updated would require two major version jumps, I decided to add a dev dependency on `standard-imghdr` and leave the docs dependency updates as a separate task.
> 
> source: https://docs.python.org/3/library/imghdr.html